composer.json

```diff
@@ -27,16 +27,16 @@
     },
     "require-dev": {
         "brain/monkey": "^2.6.1",
-        "inpsyde/php-coding-standards": "^2.0.0",
-        "roots/wordpress-no-content": "@dev",
+        "inpsyde/wp-stubs-versions": "dev-latest",
         "mikey179/vfsstream": "^v1.6.11",
-        "phpunit/phpunit": "^9.6.19",
         "phpstan/phpstan": "^2.1.1",
+        "phpstan/phpstan-deprecation-rules": "^2.0.1",
         "phpstan/phpstan-mockery": "^2.0.0",
         "phpstan/phpstan-phpunit": "^2.0.4",
+        "phpunit/phpunit": "^9.6.19",
+        "roots/wordpress-no-content": "@dev",
```

Comment:
We should check if we still need this dependency when we load stubs. So far, it is loaded in `tests/boot.php`, and `WP_Error::class` is checked in `PackageTest`-class. 🤔

Reply:
All that's needed is the `WP_Error` class, yes. Removed the package, and updated the test to use a custom `alias:` mock via Mockery.